src/contents/articles/hello-world/index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Hello World
+title: Introducing Namespace
 author: Marcus Sanatan
 date: 2015-05-24
 template: article.jade
 ---
 
-### This is **namespace**
```

Comment:
I removed this because it was competing with the blog title for the title of the post.